### PR TITLE
Fix Quarkus IT (zip) tests

### PR DESCRIPTION
**Pull request by mabartos**

### CODE REVIEW

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
 @LegacyStore
 public class QuarkusPropertiesDistTest {
 
-    private static final String QUARKUS_BUILDTIME_HIBERNATE_METRICS_KEY = "quarkus.hibernate-orm.metrics.enabled";
+    private static final String QUARKUS_BUILDTIME_HIBERNATE_METRICS_KEY = "quarkus.datasource.metrics.enabled";
```
Comment (mabartos) [inline quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java:48]: Cannot use `quarkus.hibernate-orm.*` props

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ImportDistTest.java [hunk 2]
```diff
@@ -42,6 +42,6 @@ void testImport(KeycloakDistribution dist) {
         cliResult.assertMessage("Realm 'master' imported");
         cliResult.assertMessage("Import finished successfully");
         cliResult.assertNoMessage("Changes detected in configuration");
-        cliResult.assertNoMessage("Listening on:");
+        cliResult.assertNoMessage("Listening on: http");
```
Comment (mabartos) [inline quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ImportDistTest.java:45]: Change on Quarkus side, as the `Listening on:` is present, but no URL is specified, so no HTTP connections should not be accepted.

**Comment by mabartos** [comment]
@vmuzikar The Quarkus IT(zip) tests passed.

See: https://github.com/keycloak/keycloak/actions/runs/4699109757/jobs/8332473039

**Review by vmuzikar — APPROVED** [review]
@mabartos Thanks!

**Pull request merged**